pkgs/development/python-modules/matrixzulipbridge/default.nix

```diff
@@ -0,0 +1,25 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  pythonOlder,
+}:
+buildPythonPackage rec {
+  pname = "MatrixZulipBridge";
+  version = "0.4.1";
+
+  disabled = pythonOlder "3.10";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = lib.fakeSha256;
+  };
+
+  meta = {
+    description = "A puppeting Matrix - Zulip bridge";
+    homepage = "https://github.com/GearKite/MatrixZulipBridge";
+    changelog = "https://github.com/GearKite/MatrixZulipBridge/releases/";
+    license = lib.licenses.agpl3;
```

Comment:
```suggestion
    sourceProvenance = with lib.sourceTypes; [ binaryBytecode ];
    license = lib.licenses.agpl3;
```

Reply:
You also need to add an entry to `pkgs/top-level/python-packages.nix` before this will build.